rushstack-internal-rigs/heft-rushstack-internal-node-rig/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "heft-rushstack-internal-node-rig",
+  "version": "0.1.0",
+  "description": "A rig package for Node.js projects that build using Heft inside the RushStack repository.",
```

Comment:
Add a README.md file explaining that this is used locally to manage projects in the Rush Stack repo.  Doing so is a stereotypical practice that we should encourage. Even though your rig project is not published to NPM, it is an example that the community can copy and reuse in their own repos. 

As such, I don't really think of it as "internal" (hidden and subject to change at any time), but more like "local" (customized for this repo specifically).